modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/nhttp/HttpCoreNIOSender.java

```diff
@@ -153,12 +153,9 @@ public void init(ConfigurationContext cfgCtx, TransportOutDescription transportO
 
         connpool = new ConnectionPool();
 
-        proxyConfig = new ProxyConfigBuilder().parse(transportOut).build();
-        if (log.isInfoEnabled() && proxyConfig.getProxy() != null) {
-            log.info("HTTP Sender using Proxy " + proxyConfig.getProxy() + " bypassing " + 
-                proxyConfig.getProxyBypass());
-        }
-        
+        proxyConfig = new ProxyConfigBuilder().build(transportOut);
+        log.info(proxyConfig.logProxyConfig());
+
         Parameter param = transportOut.getParameter("warnOnHTTP500");
```

Comment:
True, but the usual expectation is that, the one who modifies the code should not focus only on the changes they themselves have done. If the developer is attentive enough, he would have spotted this himself and taken the responsibility to improve these additional bits as well. :)